cpp/src/aztec/benchmark/honk_bench/compare_honk_bench.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# This script is used to compare the results of honk_bench between baseline (master) and
+# the branch from which the script is run. Simply check out the branch of interest, ensure 
+# it is up to date with local master, and run the script.
+
+# Note: This script requires that scipy be installed. This can be done via the command:
+# pip3 install -r barretenberg/cpp/build/_deps/benchmark-src/requirements.txt
```

Comment:
Maybe just do this here (probably with a particular version number)? There's no cost to doing this if it's already installed--just add `pip3 install scipy==1.5.4`.

Reply:
I've added this in as part of the script but keeping it as requirements.txt since this could plausibly be updated in the repo the future (numpy version was changed 7 months ago, for example)

Reply:
Right, that is better.